opat/NVDA-2018.yaml

```diff
@@ -0,0 +1,1364 @@
+      - num: "1.2.7"
+        components:
+          - name: "web"
+            adherence:
+              level: "does-not-support"
+              notes: "All essential information is provided. Video is not paused for additional information such as description of images."
+      - num: "1.2.8"
+        components:
+          - name: "web"
+            adherence:
+              level: "supports"
+      - num: "1.2.9"
+        components:
+          - name: "web"
+            adherence:
+              level: "supports"
+      - num: "1.4.6"
+        components:
+          - name: "web"
+            adherence:
+              level: "supports"
+      - num: "1.4.7"
+        components:
+          - name: "web"
+            adherence:
+              level: "supports"
+      - num: "1.4.8"
+        components:
+          - name: "web"
+            adherence:
+              level: "partially-supports"
+              notes: "Size and colour can not be adjusted internally."
+      - num: "1.4.9"
+        components:
+          - name: "web"
+            adherence:
+              level: "supports"
+      - num: "2.1.3"
+        components:
+          - name: "web"
+            adherence:
+              level: "partially-supports"
+              notes: "To have the output of some commands read as individual letters or phonetically, instead of read as sentences, some keystrokes can be pressed twice quickly (within a second). For instance, to spell the title bar instead of speak it as words."
+      - num: "2.2.3"
+        components:
+          - name: "web"
+            adherence:
+              level: "partially-supports"
+              notes: "To have the output of some commands read as individual letters or phonetically, instead of read as sentences, some keystrokes can be pressed twice quickly (within a second). For instance, to spell the title bar instead of speak it as words."
+      - num: "2.2.4"
+        components:
+          - name: "web"
+            adherence:
+              level: "supports"
+      - num: "2.2.5"
+        components:
+          - name: "web"
+            adherence:
+              level: "partially-supports"
+              notes: "The certified expert exams are time limited and so expire after the time elapses. The time for the exams are set taking into account that most users will be relying on NVDA and not necessarily as fast as a sighted user."
+      - num: "2.3.2"
+        components:
+          - name: "web"
+            adherence:
+              level: "supports"
+      - num: "2.4.8"
+        components:
+          - name: "web"
+            adherence:
+              level: "partially-supports"
+              notes: "On the web site breadcrumbs are not provided."
+          - name: "software"
+            adherence:
+              level: "supports"
+              notes: "In NVDA itself, there are no “breadcrumb trails” to follow."
```

Comment:
@mgifford quotes in quotes I think are causing the pre-commit failures. If you switch this to regular quotes and escape them it should be fine.